packages/trader/src/sass/app/modules/trading.scss

```
@@ -528,6 +528,14 @@
             align-self: flex-end;
         }
     }
+    &__trade-type-tabs {
+        border-radius: 4px;
```

```suggestion
        border-radius: $BORDER_RADIUS;
```
@hamza-deriv nit-pick )